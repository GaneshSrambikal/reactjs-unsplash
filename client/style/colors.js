import { lighten, darken } from 'polished';

export const primaryColor1 = '#111';
export const primaryColor2 = lighten(0.07, primaryColor1);
export const secondaryColor1 = '#999';
export const secondaryColor2 = darken(0.04, secondaryColor1);
export const white = '#FFFFFF';
export const borderInAvtiveColor = '#ddd';
export const borderAvtiveColor = '#999';