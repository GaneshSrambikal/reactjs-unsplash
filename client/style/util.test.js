import React from 'react';
import styled from 'styled-components';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import {media} from '../style/util';

const Div = styled.div `
  ${media.giant `
    width: 1170px;
  `}
  ${media.desktop `
    width: 992px;
  `}
  ${media.tablet `
    width: 768px;
  `}
  ${media.phone `
    width: 376px;
  `}
`;

describe('Test style Util', () => {
  describe('media function to generate CSS media query', () => {
    it('should generate the media query base on phone size', () => {
      const wrapper = renderer
        .create(<Div />)
        .toJSON();
      expect(wrapper).toMatchStyledComponentsSnapshot();
    });
  });
});