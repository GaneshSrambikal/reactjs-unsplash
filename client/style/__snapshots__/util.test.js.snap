// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test style Util media function to generate CSS media query should generate the media query base on phone size 1`] = `
@media (max-width:73.125em) {
  .fpdmRR {
    width: 1170px;
  }
}

@media (max-width:62em) {
  .fpdmRR {
    width: 992px;
  }
}

@media (max-width:48em) {
  .fpdmRR {
    width: 768px;
  }
}

@media (max-width:23.5em) {
  .fpdmRR {
    width: 376px;
  }
}

<div
  className="sc-bdVaJa fpdmRR"
/>
`;
